src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +13,9 @@
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users.stream()
+                .filter(user -> user.getName().equals(name))
```

Comment:
이미 구현되어 있는 `User.matchName()`을 사용해보면 어떨까요?
```suggestion
                .filter(user -> user.matchName(name))
```